google/cloud/internal/grpc_options.h

```diff
@@ -82,6 +84,8 @@ struct GrpcBackgroundThreadsOption {
   BackgroundThreadsFactory value;
 };
 
+}  // namespace internal
+
 namespace internal {
```

Comment:
Is it strange to close the `internal` namespace, and then immediately re-open it?

Reply:
A little strange, yes. The reason I did that is because the `internal` namespace being opened on line 89 should always remain internal, whereas the `internal` namespace closing on line 87 will be removed when these options move out of the `internal` namespace. I wanted to distinguish between the "temporarily" internal namespace and the permanent internal namespace so that we didn't accidentally publicly expose something.